### ISSUE TITLE
Geospatial information (points, polygons)

### ISSUE THREAD

**Issue by kitchenprinzessin3880** [AuScope/ckan-docker#28]
The submission form should allow the user to specify either polygon (bounding box) or point information and other relevant fields (EPSG, elevation, vertical datum) manually or by clicking on the map.
You can use CSIRO DAP as a reference when designing the component above.
<img width="982" alt="Screenshot 2024-01-25 at 4 14 16 pm" src="https://github.com/AuScope/ckan-docker/assets/11935313/7ba4a760-09b0-4caa-bb3f-854944a6cc9d">

**Comment by NTaherifar** [comment]
I have incorporated a Leaflet map into the submission form, enabling users to either pinpoint a single location or draw a bounding box based on their preference. I've also developed a script to validate latitude, longitude, and bounding box inputs. While the module is operational, it is not a high priority. I have pushed the changes to the master branch and will revisit the code for refinement at a later, more convenient time.
